OpenRA.Mods.Common/Scripting/Properties/PlayerProperties.cs

```diff
@@ -31,7 +31,7 @@ public PlayerProperties(ScriptContext context, Player player)
 		public HSLColor Color { get { return Player.Color; } }
 
 		[Desc("The player's race.")]
-		public string Race { get { return Player.PlayerReference.Race; } }
+		public string Race { get { return Player.PlayerReference.Faction; } }
```

Comment:
This may be also safe to rename as few people already use this new API. Probably in it's own pull request though.